improver_tests/test_field_texture.py

```diff
@@ -233,10 +233,10 @@ def test_metadata_coords(multi_cloud_cube):
         conventions after that plugin has completed."""
 
     result = PLUGIN.process(multi_cloud_cube)
-
     expected_units = "1"
     expected_dims = ["projection_y_coordinate", "projection_x_coordinate"]
     expected_scalar_coord = "texture_of_cloud_area_fraction"
+    expected_dtype = "float32"
```

Comment:
Perhaps it would be best to add this check to a test that is already dealing with the cube data, perhaps `test_full_process`, given that this particular test is focused on testing the coordinate metadata.

Reply:
Done.